tests/queries/0_stateless/02222_create_table_without_columns_metadata.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+
+METADATA_PATH=$($CLICKHOUSE_CLIENT -q "select metadata_path from system.databases where name = '$CLICKHOUSE_DATABASE'")
+
+$CLICKHOUSE_CLIENT -q "insert into table function file(data.jsonl, 'JSONEachRow', 'x UInt32 default 42, y String') select number as x, 'String' as y from numbers(10)"
+
+$CLICKHOUSE_CLIENT -q "drop table if exists test"
+$CLICKHOUSE_CLIENT -q "create table test engine=File(JSONEachRow, 'data.jsonl')"
+
+cat $METADATA_PATH/test.sql | grep -v "UUID"
```

Comment:
It's a bit tricky, because path may be relative and user may not have access to read the file, but it's ok if test works in CI. Also we can try `SHOW CREATE` query and `DETACH/ATTACH` after removing `data.jsonl`. Btw, is it supposed to be `data.json` without `l`?

Reply:
Your suggestion about the test is much better than current way, I will change it, thanks. `jsonl` is one of extensions for `JSONEachRow` format.